package.json

```diff
@@ -397,6 +407,10 @@
           "when": "editorLangId == sas && !SAS.hideRunMenuItem",
           "command": "SAS.run"
         },
+        {
```

Comment:
looks like duplicated?

Reply:
Yeah, Not sure how that got in there. I removed it.